src/views/Coach/EditProfile/index.tsx

```diff
@@ -146,324 +160,364 @@ export const EditProfile = ({ currentUser, setCurrentUser }) => {
+                        Public Bio
+                      </Heading>
+                    </Box>
+                    <Box flex={1}>
+                      <FormControl
+                        isRequired
+                        isInvalid={props.touched.bio && !!props.errors.bio}
+                      >
+                        <FormLabel htmlFor="bio">
+                          Please limit to one paragraph. You can write about your
+                          years of experience, industry, or skills.
+                        </FormLabel>
+                        <Textarea
+                          id="bio"
+                          name="bio"
+                          value={props.values.bio}
+                          onChange={props.handleChange}
+                          onBlur={props.handleBlur}
+                          placeholder="John resides in Nashville, TN where he loves to fish, golf, spend time with this family, and play the latest video games. He prides himself on human connection."
+                        />
+                        <FormErrorMessage>{props.errors.bio}</FormErrorMessage>
+                      </FormControl>
+                    </Box>
+                    <Box flexBasis="100%" marginTop={10} marginBottom={6}>
+                      <Heading as="label" size="md" fontWeight="normal">
+                        Change Password
+                      </Heading>
+                      <Box
+                        w="60%"
+                      >
+                        <FormControl
+                          isInvalid={isTouchedPassword && !!props.errors.retype_password}
+                        >
+                          <Flex
+                            mt={4}
+                          >
+                            <Input
+                              mr={4}
+                              variant="outline"
+                              id="update_password"
+                              name="update_password"
+                              value={props.values.password}
+                              onChange={props.handleChange}
+                              onBlur={props.handleBlur}
+                              placeholder="Password"
+                            />
+                            <Input
+                              variant="outline"
+                              id="retype_password"
+                              name="retype_password"
+                              onChange={props.handleChange}
+                              onBlur={props.handleBlur}
+                              placeholder="Retype Password"
+                            />
+                          </Flex>
+                          <Center>
+                            <FormErrorMessage>{props.errors.retype_password}</FormErrorMessage>
+                          </Center>
+                        </FormControl>
+                      </Box>
```

Comment:
This 2 of 2 changes made to the form. The rest of the form is unchanged, no need to parse through what looks like a massive diff.